### ISSUE TITLE
Documentation does not specify how to include unsub links in HTML

### ISSUE THREAD

**Issue by kittytfg** [sendgrid/docs#6239]
## Expected content
Replacement Tag - Your custom-defined tag for your templates. Use this to place your unsubscribe URL wherever you want in your template. If you provide this, make sure you put it within an <a href= "your-tag"> tag. You do not need to include brackets in HTML. If you don’t provide this, your default unsubscribe content will be automatically added to the bottom of your email.

_It is not made clear in the actual content how to include the replacement tag. In other documentation it suggests using <a href ="[your-tag]"> which is incorrect_

## Actual content
Replacement Tag - Your custom defined tag for your templates. Use this to place your unsubscribe URL wherever you want in your template. If you provide this, make sure you put it within an <a href= tag. If you don’t provide this, your default unsubscribe content will be automatically added to the bottom of your email.

**Link to page**: https://sendgrid.com/docs/ui/account-and-settings/tracking/#subscription-tracking

**Comment by reinisla** [comment]
Agree, tried like 10 times until I got it working. I had a long a tag with href="{{{unsubscribe}}}" and I had to add {{{unsubscribe}}} as Replacement tag

**Comment by Biboswan** [comment]
The overall documentation on unsub links should be made clear with complete examples.
